linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -129,6 +129,12 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_LINUX_07_010: [** `insert_idx` and `consume_idx` for the task array shall be initialized to 0. **]**
 
+**S_R_S_THREADPOOL_LINUX_11_001: [** `threadpool_create` shall initialize internal threapool data items **]**
+
+**S_R_S_THREADPOOL_LINUX_07_020: [** `threadpool_create` shall create number of `min_thread_count` threads for `threadpool` using `ThreadAPI_Create`. **]**
```

Comment:
>create number  [](http://example.com/codeflow?start=63&length=14)

I remember I had a comment about a missing *a*.
And the comment is marked as Done, but the fix is not here.

Did you forget to push?

Reply:
I think it got reversed in some Ctrl + Z for something else. Sorry about that.